CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 A new type-level synopsis structure in the meta-index now massively
+  speeds up string queries with very few results.
+  [#1214](https://github.com/tenzir/vast/pull/1214)
```

Comment:
This is another case where I don't see the need for a "feature" changelog. It's a non-functional change.

It would only make sense to introduce a performance-related category where we document performance-affecting changes.